app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -58,7 +58,7 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), INews
         })
         vNewsRecycler.addOnItemClickListener(object : OnItemClickListener {
             override fun onItemClicked(position: Int, view: View) {
-                presenter.onSelectedItem(position)
+                presenter.onSelectedItem(position, (view.findViewById(R.id.vNewLikeIcon) as ImageView))
```

Comment:
It's necessary to use findViewById and cast it to ImageView? Have you tried to use directly vNewLikeIcon? However, I don't think that passing a View to the presenter to it's a good idea. Maybe the "extras" var that you create on presenter could be done on the fragment.

Reply:
I pass the reference of the view, for sharedElement transition to be done with the recycler items